Source/Classes/PINRemoteImageManager.m

```diff
@@ -281,6 +282,16 @@ - (void)setValue:(nullable NSString *)value forHTTPHeaderField:(nullable NSStrin
     });
 }
 
+- (void)setRequestConfiguration:(PINRemoteImageManagerRequestConfigurationHandler)configurationBlock {
```

Comment:
The `requestConfigurationHandler` property isn't exposed publicly, only the `setRequestConfiguration:` method.